src/saving/SaveHelper.cs

```diff
@@ -121,6 +142,7 @@ public static void LoadSave(string name)
     {
         GD.Print("Starting load of save: ", name);
         new InProgressLoad(name).Start();
+        ResetSaveFreshness();
```

Comment:
I think this should do the *opposite* if you have just loaded a save and quit, the effect is basically the same as having saved just recently, so no warning about losing progress is necessary in this case.